webpack.config.js

```diff
@@ -17,17 +17,19 @@ module.exports = {
         rules: [
             {
                 test: /\.ts$/,
-                loader: require.resolve("awesome-typescript-loader"),
-                options: {
-                    configFileName: "./src/tsconfig.json",
-                    useCache: true,
-                    declaration: false
-                }
+                use: [
+                    {
+                        loader: 'ts-loader',
+                        options: {
+                            configFile: "./tsconfig.json",
+                        }
+                    }
+                ],
             }
         ]
     },
     plugins: [
-        new webpack.IgnorePlugin(/^electron$/),
-        new webpack.BannerPlugin({ banner: "#!/usr/bin/env node", raw: true})
+        new webpack.IgnorePlugin({ resourceRegExp: /^electron$/ }),
```

Comment:
unsure how to test this?

Reply:
I don't think this actually does anything (anymore). If you comment this line out on `develop` we generate exactly the same bundle:  

```
# Before
> sha1sum dist/bin/conjure-typescript
42684b71a895a8bfedb679740388454a3b10f1fa  dist/bin/conjure-typescript
```

```
# After
 > sha1sum dist/bin/conjure-typescript
42684b71a895a8bfedb679740388454a3b10f1fa  dist/bin/conjure-typescript
```